mne/viz/topomap.py

```diff
@@ -63,6 +63,9 @@ def _prepare_topo_plot(inst, ch_type, layout):
         if ch_type == 'eeg':
             picks = pick_types(info, meg=False, eeg=True, ref_meg=False,
                                exclude='bads')
+        elif ch_type == 'ref_meg':
+            picks = pick_types(info, meg=False, ref_meg=True,
+                               exclude='bads')
```

Comment:
This will presumably change `topo` plots for any system with reference channels. We need to check to see if it still looks okay, and probably provide a way to turn it off here, too, e.g. a `ref_meg=False` kwarg for the `plot_topo`-class of functions.